lite2/setup.go

```diff
@@ -0,0 +1,69 @@
+package lite
+
+import (
+	"time"
+
+	"github.com/tendermint/tendermint/lite2/provider"
+	"github.com/tendermint/tendermint/lite2/provider/http"
+	"github.com/tendermint/tendermint/lite2/store"
+)
+
+// NewNetClient and NewNetClientFromTrustedStore each initiate an instance of a lite client using just RPC addresses
+// to form the providers to the lite client
+func NewNetClient(
```

Comment:
On the second thought, `Net` is probably too abstract as it can represent both HTTP and P2P. What do you think about `NewHTTPClient`?

Reply:
Sure sounds good